.github/workflows/main.yml

```diff
@@ -536,6 +536,11 @@ jobs:
     - name: Run Python unit tests (Ubuntu ${{ matrix.UbuntuVersion }})
       timeout-minutes: 30
       run: docker exec mlos-build-python-${{ matrix.UbuntuVersion }} make python-test
+    - name: Upload coverage to codecov
+      uses: codecov/codecov-action@v1
+      with:
+        file: ./coverage.xml
+        flags: unittests
```

Comment:
```suggestion
        flags: unittests
        fail_ci_if_error: true
```

Reply:
Undecided about this.  Seems more useful if we're providing a badge, but then we're dependent upon an external service.